cdk/v2/handler.js

```diff
@@ -0,0 +1,29 @@
+const { WorkflowUtils, WorkflowEngine } = require("rudder-workflow-engine");
+const {
+  getRootPathForDestination,
+  getWorkflowPath,
+  getPlatformBindingsPaths
+} = require("./utils");
+
+const destinationWorkflowEngineMap = new Map();
+
+function getWorkflowEngine(destName) {
```

Comment:
destination can have more than one workflow so how do we handle that?
We can come up some standard way of naming these.

Reply:
Addressed.